universal-application-tool-0.0.1/app/services/cloud/azure/BlobStorage.java

```diff
@@ -121,7 +123,7 @@ public StorageServiceName getStorageServiceName() {
     NullClient() {}
 
     @Override
-    public String getSasToken(String fileName) {
+    public String getSasToken(String fileName, Optional<String> userFileName) {
```

Comment:
sorry my java knowledge might be off here but does this and the one below it not overlap?

Reply:
They're both overrides of the same method in different inner classes. One's the dev client (talks to emulator) and the other is the prod client (talks to real storage instance)